python/invoke_release/tasks.py

```diff
@@ -1503,3 +1512,22 @@ def wheel(_):
         archive_name=archive_name,
         base_dir=base_dir
     ))
+
+
+def create_pull_request(options, user, token, title,
+                        github_url='https://api.github.com'):
+    headers = {"Authorization": "token %s" % token,
+            'Content-Type': 'application/json'}
+    data = {
+            "title": title,
+            "base": options.base,
+            "head": options.head
+    }
+    if options.body:
+        data["body"] = options.body
+    response = requests.post(GITHUB_PULL_URL % github_url, headers=headers,
```

Comment:
Same here about the `%` syntax. `GITHUB_PULL_URL` should be `https://api.github.com/repos/{repo_name}/pulls` and use here `GITHUB_PULL_URL.format(repo_name=<repository_name>)`.